adbe/main.py

```diff
@@ -274,7 +274,8 @@ def main():
     elif args['apps'] and args['list'] and args['system']:
         adb_enhanced.list_system_apps()
     elif args['apps'] and args['list'] and args['third-party']:
-        adb_enhanced.list_non_system_apps()
+        packages = adb_enhanced.get_list_non_system_apps()
+        print_message('\n'.join(packages))
```

Comment:
It is preferable not to do it here even if that means a 2-line function in adb_enhanced. Let's keep the main.py as small as possible. I might move to a different flag parsing library in the future.